### ISSUE TITLE
IE8 Support

### ISSUE THREAD

**Issue by jconstan** [mbkom/project-netvide#13]
Can you take a look at the Beta UI site and explain what we've managed to get wrong when implememting your IE8 support, Thanks John

![capture](https://cloud.githubusercontent.com/assets/551874/9740438/a69a7394-564c-11e5-9070-9627b9b36aa2.PNG)

**Comment by bahrul221** [comment]
try add the following code in the head tag :

**Comment by Teletrak** [comment]
I had already implemented the above code, so problem is being caused by something else?

**Comment by bahrul221** [comment]
I don't think so, it's work on demo page, try put the code above after loading all the css and javascript files, or before closing head tag &lt;/head&gt;. please check the demo page http://demo.mbkom.com/08/netvide/dashboard.html

**Comment by Teletrak** [comment]
I will try that.
Thank you.

**Comment by Teletrak** [comment]
I have found out what was causing this issue.
In our implementation we were getting our bootstrap css files from a CDN, and respond.js doesn't natively handle cross domain css requests.
See https://github.com/scottjehl/Respond/blob/master/README.md#cdnx-domain-setup

**Comment by bahrul221** [comment]
you are right, you need to upload a proxy page

**Comment by Teletrak** [comment]
Hi Bahrul
Do you have any experience of uploading the proxy page?
(We would prefer to use a CDN to get Bootstrap)

**Comment by bahrul221** [comment]
I've pushed on github, please check dashboard.html how I use a proxy page
